configs/compute/1gpu.cfg

```diff
@@ -1,11 +1,11 @@
-numGameThreads = 128
+numGameThreads = 256
```

Comment:
Is this better for all models or just for ViT models? If latter, do we need to make it conditional on model type? If former, perhaps we should split this into a separate PR?